modules/ding_innskraning/ding_innskraning.module

```diff
@@ -0,0 +1,780 @@
+<?php
+
+use RobRichards\XMLSecLibs\XMLSecEnc;
+use RobRichards\XMLSecLibs\XMLSecurityDSig;
+
+/**
+ * @file
+ * Provides login via https://innskraning.island.is.
+ *
+ * Forked from xi_innskraning.
+ */
+
+/**
+ * Implements hook_permission().
+ *
+ * Adds permission to allow users to administer SSO login.
+ */
+function ding_innskraning_permission() {
+  return array(
+    'administer ding innskraning' => array(
+      'title' => t('Administer Ding Innskraning'),
+      'description' => t('Perform administration tasks for Ding Innskraning module.'),
+    ),
+  );
+}
+
+/**
+ * Implements hook_menu().
+ *
+ * Provides end point for time tracking.
+ */
+function ding_innskraning_menu() {
+  $items = array();
+
+  $items['admin/config/ding/ding-innskraning'] = array(
+    'title' => 'Ding Innskraning settings',
+    'description' => 'Settings for Ding Innskraning module',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('ding_innskraning_settings'),
+    'access arguments' => array('administer ding innskraning'),
+    'type' => MENU_NORMAL_ITEM,
+    'file' => 'inc/ding_innskraning.admin.inc',
+  );
+
+  $items['saml/login'] = array(
```

Comment:
Nitpicking but I would normally expected that modules "namespaced" their menu entries e.g. `ding_innskraning/saml/login`.

Reply:
Yeah, but menu entries is also publicly visible, so often they're not. But `saml` aren't good either.

I've changed it to `innskraning/login`.